### PR TITLE
Apollo Lake needs MCLK 1 for working da7219

### PR DESCRIPTION
**Pull request by coolstar**
Headphone output requires MCLK_ID to be set to 1

Fix for headphone issue noticed in #6963

Tested: https://www.youtube.com/watch?v=mtPHPJKPp6o

### CODE REVIEW

**Comment by sofci** [comment]
Can one of the admins verify this patch?
> reply **test&nbsp;this&nbsp;please** to run this test once

**Comment by gkbldcig** [comment]
Can one of the admins verify this patch?

File: tools/topology/topology1/sof-apl-da7219.m4 [hunk 1]
```diff
@@ -157,21 +157,21 @@ PCM_PLAYBACK_ADD(HDMI3, 7, PIPELINE_PCM_7)
 # BE configurations - overrides config in ACPI if present
 #
 
-#SSP 5 (ID: 0) with 19.2 MHz mclk with MCLK_ID 0 (unused), 1.536 MHz blck
+#SSP 5 (ID: 0) with 19.2 MHz mclk with MCLK_ID 1 (unused), 1.536 MHz blck
```
Comment (lyakh) [inline tools/topology/topology1/sof-apl-da7219.m4:160]: if it *has* to be 1, then it's probably not "unused?"

Comment (coolstar) [reply tools/topology/topology1/sof-apl-da7219.m4:160]: GLK needs to be updated too in that case, since it also has to be 1 there (and is already)

Comment (plbossart) [reply tools/topology/topology1/sof-apl-da7219.m4:160]: no, I think the comment was intentional. The MCLK is only relevant for the headset codec IIRC, it's not used by the amplifier.
and while in theory we could use 2 MCLKs for the two different SSPs, I am not sure this was ever tested.

Comment (plbossart) [reply tools/topology/topology1/sof-apl-da7219.m4:160]: the topology uses SSP5 (speaker - maxim98357a, BE link 0) so this is correct and doesn't need a change.

**Review by plbossart — APPROVED** [review]
LGTM thanks @coolstar

IIRC I could never get the headset detection to work on my Reef device, maybe that was the problem. I bricked that device so can't test this unfortunately, that was the first SOF device to ever be tested... 2018?

**Review by kv2019i — APPROVED** [review]

**Review by kv2019i — APPROVED** [review]
Thanks @coolstar , looks good to go!

**Comment by kv2019i** [comment]
SOFCI TEST

**Pull request merged**